src/client/Microsoft.Identity.Client/AuthenticationResult.cs

```diff
@@ -196,9 +196,10 @@ internal AuthenticationResult(
                 authenticationScheme.FormatResult(this);
             });
 
-            if (authenticationScheme.TelemetryTokenType == 5)
+            if (!authenticationScheme.TelemetryTokenType.Equals("Bearer"))
```

Comment:
nit: string equality should generally be case insensitive in MSAL. If you want case sensitive just use ==

Why don't we just always record this, even for Bearer?

Reply:
Discussed offline, will switch to always record data for all token types